### DISCUSSION TITLE
Create User Schema:  /opt/jans/jans-cli/scim-cli.py not found

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4811] [Q&A] [answered]
Hello there,

I was trying to get the schema for user creation by executing the cli- command **/opt/jans/jans-cli/scim-cli.py --schema /components/schemas/UserResource**
But the command-file was not there in the directory **/opt/jans/jans-cli/**
Please suggest how should I proceed further on this.

**Comment by imShakil** [comment]
Hi, dhanan-np!

It's outdated documentation. There is no scim-cli.py anymore. Either You can use `jans-cli-tui.py` or `config-cli.py` to get details on it. 
Example: 
```
/opt/jans/jans-cli/config-cli.py --info ScimConfigManagement
```

**Reply by imShakil** [reply, marked as answer]
<img width="1441" alt="Screenshot 2023-05-08 at 12 58 06 PM" src="https://user-images.githubusercontent.com/20867846/236756424-fc93a109-8e08-4977-a49d-e59b4ac0f60a.png">

**Reply by dhanan-np** [reply]
I got the schema like this for create user-

```
{
  "dn": "string",
  "userId": "string",
  "updatedAt": "string",
  "createdAt": "string",
  "oxAuthPersistentJwt": [
    "string"
  ],
  "customObjectClasses": [
    "string"
  ],
  "inum": "string",
  "mail": "string",
  "displayName": "string",
  "jansStatus": "string",
  "givenName": "string",
  "userPassword": "string",
  "status": "string",
  "baseDn": "string"
}
```

Which is not including any of the custom attributes.
I was expecting that it will respond with all the fields that I had enabled for user-creation.

**Comment by ossdhaval** [comment]
Tagging @devrimyatar for comments.
